design/fast_finality.md

```diff
@@ -0,0 +1,273 @@
+# Fast finality design
+
+## Scope
+
+The introduction of a single sequencer to improve cost-per-transaction during the bootstrapping phase, as 
+described in the [Bootstrapping Strategy design doc](./Bootstrapping_strategy.md).
+
+## Glossary
+
+* *Light batch*: A set of transactions produced by the designated sequencer, considered to be soft-finalised in that 
+  order
+
+## Requirements
+
+* Finality
+  * Transaction *soft* finality (finality guaranteed by the sequencer) is achieved in under one second
+  * There is eventual transaction *hard* finality (finality guaranteed by the L1)
+  * The sequencer is incentivised to achieve hard finality on an agreed cadence
```

Comment:
no mention in the design of what the incentive is.
Give, during bootstrapping it's a central sequencer operated by the foundation, it wants everything to run smoothly, and as cheaply as possible

Reply:
Added.